xbmc/GUIInfoManager.cpp

```diff
@@ -2118,6 +2118,15 @@ const infomap videoplayer[] =    {{ "title",            VIDEOPLAYER_TITLE },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`VideoFilter`</b>,
+///                  \anchor RetroPlayer_VideoFilter
```

Comment:
this breaks our pretty doxygen table of infolabels. When creating instances of custom doxygen commands (such as \table_row3) every comma is interpreted as a new entrance (in this case a new column). They need to be escaped otherwise you break the table for the infolabels that follows. See example below and note the escaped commas in "Nearest-neighbor" and "Bilinear"

```
///   \table_row3{   <b>`VideoFilter`</b>,
///                  \anchor RetroPlayer_VideoFilter
///                  _string_,
///     Returns the video filter of the currently-playing game.\n
///     The following values are possible:
///     - nearest (Nearest-neighbor\, i.e. pixellate)
///     - linear (Bilinear\, i.e. smooth blur)
///     - <path to shader> (not supported yet)
///   }
```